dev-java/mockito/mockito-2.28.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="org.mockito:mockito-core:${PV}"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Mockito mock objects library core API and implementation"
+HOMEPAGE="https://github.com/mockito/mockito"
+SRC_URI="https://github.com/mockito/mockito/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P}"
+
+LICENSE="MIT"
+SLOT="2"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+
+CP_DEPEND="
+	dev-java/asm:9
+	dev-java/byte-buddy:0
+	dev-java/objenesis:0
+"
+
+DEPEND="
+	dev-java/junit:4
+	dev-java/opentest4j:0
+	>=virtual/jdk-1.8:*
+	${CP_DEPEND}
+	test? ( dev-java/assertj-core:3 )
+"
+
+RDEPEND="
+	>=virtual/jre-1.8:*
+	${CP_DEPEND}
+"
+
+PATCHES=( "${FILESDIR}/mockito-2.28.2-skip-failing-tests.patch" )
```

Comment:
it would be good to mention somewhere what the failures are and what is the cause, if known.

Reply:
I've put the complete build log with test failures in https://bugs.gentoo.org/903897